lxd/resources/system.go

```diff
@@ -124,6 +135,94 @@ func GetSystem() (*api.ResourcesSystem, error) {
 	return &system, nil
 }
 
+// getSystemFromLshw gets the system information from the `lshw` command.
+func getSystemFromLshw() (*api.ResourcesSystem, error) {
+	var buf bytes.Buffer
+
+	err := shared.RunCommandWithFds(context.TODO(), nil, &buf, "lshw", "-json")
```

Comment:
Is lshw in the snap?

Reply:
Oh right. No, it's not in the LXD snap.

Reply:
Please can we add that as a PR then.

Reply:
https://github.com/canonical/lxd-pkg-snap/pull/140